### DISCUSSION TITLE
How to update on the 3rd day of the month?

### DISCUSSION THREAD

**Discussion by hugovk** [renovatebot/renovate#19513] [Request Help]
In our [`renovate.json`](https://github.com/python-pillow/Pillow/blob/2f3561fe33df96cdb62d66f387009837ebf7254e/.github/renovate.json#L16) we have:

```json
    "schedule": ["on the 3rd day of the month"]
```

But Renovate PRs are being created immediately, for example:

* https://github.com/python-pillow/Pillow/pull/6776
* https://github.com/python-pillow/Pillow/pull/6806
* https://github.com/python-pillow/Pillow/pull/6813

I understand [Later.js](https://bunkat.github.io/later/) is used for scheduling, and if I run this code at https://npm.runkit.com/:

```js
var later = require('later');
var sched = later.parse.text('on the 3rd day of the month'),
      occurrences = later.schedule(sched).next(10);

for(var i = 0; i < 5; i++) {
  console.log(occurrences[i]);
}
```

I get the expected output for the third day of each month:

```
Tue Jan 03 2023 02:00:00 GMT+0200 (Eastern European Standard Time)
Fri Feb 03 2023 02:00:00 GMT+0200 (Eastern European Standard Time)
Fri Mar 03 2023 02:00:00 GMT+0200 (Eastern European Standard Time)
Mon Apr 03 2023 03:00:00 GMT+0300 (Eastern European Summer Time)
Wed May 03 2023 03:00:00 GMT+0300 (Eastern European Summer Time)
```

Am I doing something wrong? Thanks!

**Comment by viceice** [comment]
schedule is for branch creation, not PR creation. so those branches probably exists before your config change.

**Reply by hugovk** [reply]
These branches were all called `renovate/github-actions` and the bot deleted the branch right after merge.

For example: https://github.com/python-pillow/Pillow/pull/6813

But do you mean a branch will have been created on the third, but if there's no changes, it just sits there until one turns up?

Is there a way to configure when the PRs will be created?

**Comment by rarkins** [comment]
You can also choose cron scheduling instead, if you want to be sure that the input is correct

**Reply by hugovk** [reply]
Thanks, will give [`0 12 3 * *`](https://crontab.guru/#0_12_3_*_*) a shot.
